src/Tickets/Commerce/Admin/Featured_Settings.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace TEC\Tickets\Commerce\Admin;
+
+use \Tribe__Template;
+
+/**
+ * Featured Settings for TC Payments Tab.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Admin
+ */
+class Featured_Settings {
+	
+	/**
+	 * Stores the instance of the template engine that we will use for rendering the elements.
+	 *
+	 * @since TBD
+	 *
+	 * @var Tribe__Template
+	 */
+	protected $template;
+	
+	/**
+	 * Gets the template instance used to setup the rendering html.
+	 *
+	 * @since TBD
+	 *
+	 * @return Tribe__Template
+	 */
+	public function get_template() {
+		if ( empty( $this->template ) ) {
+			$this->template = new \Tribe__Template();
+			$this->template->set_template_origin( \Tribe__Tickets__Main::instance() );
+			$this->template->set_template_folder( 'src/admin-views/settings/featured' );
+			$this->template->set_template_context_extract( true );
+			$this->template->set_template_folder_lookup( true );
```

Comment:
When using a admin view, we dont want to look into theme for overwrites.
```suggestion
			$this->template->set_template_folder_lookup( false );
```